src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {movies.map(movie => <MovieCard key={movie.imdbId} movie={movie} />)}
```

Comment:
1 line is good but u should to it more readable

```suggestion
      {movies.map(movie => (
        <MovieCard key={movie.imdbId} movie={movie} />
      )}
```